README.md

```diff
@@ -28,94 +62,470 @@ Run the installation command:
+affinidi logout --help
+```
+
+ ### **affinidi show**
+This command displays the details of a resource. The current available resource types are:
+- project
+- schema
+
+USAGE
+```
+$ affinidi show [SUBCOMMAND] [ARG...] [FLAGS]
+```
+
+SUBCOMMANDS
+```
+project         Shows information about a given project
+schema          Shows the details of a schema
+```
+
+FLAGS
+```
+--output json     Overrides default plain text view and shows details in json format
+```
+
+To show a project:
+```
+$ affinidi show [<project-id>] 
+```
+if you simply type this, the CLI will prompt you to choose from a list of available projects: 
+```
+$ affinidi show project
+```
+To show a schema:
+```
+$ affinidi show schema [<schema-id>] [--output json]
+```
+
+EXAMPLES
+```
+$ affinidi show project example-id
+$ affinidi show schema example-id
+  ```
+
+You can also see the help for the command in the CLI:
+```
+$ affinidi show --help
+$ affinidi show project --help
+$ affinidi show schema --help
+```
+  ### **affinidi sign-up**
+Create an Affinidi account with this command to use our privacy preserving tools. You will need your email adress, and then confirm the authentication with the code sent to your email.
+
+USAGE
+```
+$ affinidi sign-up [EMAIL]
+```
+
+You can also simply type this and follow the prompts: 
+```
+$ affinidi sign-up
+```
+
+You can also see the help for the command in the CLI:
+```
+$ affinidi sign-up --help
+```
+
+ ### **affinidi use**
+The Use command lets you choose and activate a project. An active project is a prerequisite for executing most commands.
+
+USAGE
+```
+$ affinidi use project [<project-id>] [FLAGS]
+```
 
-### How to create a schema
+FLAGS
+```
+-o, --output=(json|json-file)  [default: json] print details of the project to use as JSON
+```  
 
-#### 1 Prepare a json file with schema details:
+
+EXAMPLES    
+```
+    $ affinidi use project example-id
+```
+
+You can also see the help for the command in the CLI:
+```
+$ affinidi use --help
+$ affinidi use project --help
+```
+
+ ### **affinidi verify-vc**
+Verifies a verifiable credential
+
+USAGE
+```
+$ affinidi verify-vc -d <value>
+```
+FLAGS
+```
+-d, --data=<value>  (required) Source json file with credentials to be verified
+```
+EXAMPLES
+```
+$ affinidi verify-vc -d "/exampleVc.json"
+```
+
+You can also see the help for the command in the CLI:
+```
+$ affinidi verify-vc --help
+```
+&nbsp;
+
+&nbsp;
+
+
+#
+## About Schemas and Verifiable Credentials
+
+Schemas are representations of the properties that define a VC. They are a composite of [JSON Schema](https://json-schema.org/specification.html), [JSON-LD](https://www.w3.org/TR/json-ld11/) context and metadata (description, version and ownership). You can use Affinidi's [Schema Manager](https://affinidi-schema-manager.prod.affinity-project.org/api-docs) to find the right schema for your verifiable credential or to create a new one – either on the basis of an already existing schema,
+or completely from scratch. 
```

Comment:
can you search for schemas in the UI? I couldn't find a way to do it --- we are suggesting the use of the Schema Manager api to search for schemas because all other options are very cumbersome (including the CLI `affinidi list schemas` command)

Reply:
Limitation of the Extension is that we are only showing Schemas that have been created for my project. This limitation doesn't exist in the CLI. As Console is currently closed, I understand why Vitaly is proposing to use standalone schema manager but I would prefer not to link to that one as we will possibly deprecate it. And as Diego pointed out the standalone doesn't have search functionality either. I suggest to recommend using the API or CLI for now that at least offer some query functionality.